app/components/funding_widget_component.html.erb

```diff
@@ -5,26 +5,39 @@
     <div class="call-to-action__text">
       <div class="call-to-action__form">
         <%= form_with(model: form, url: path, method: :get, local: true, html: { data: { "controller": "funding-widget", "funding-widget-target": 'form'}}) do |f| %>
+          <% if form.errors.any? %>
+            <div class="govuk-error-summary" data-module="govuk-error-summary">
```

Comment:
Does the error summary have to go at the top of the page, rather than just within this element?

Might be a question for @garethwilliamsdesign